src/bionics.cpp

```diff
@@ -2820,8 +2820,40 @@ void bionic::toggle_safe_fuel_mod()
     }
     if( !has_flag( flag_SAFE_FUEL_OFF ) ) {
         set_flag( flag_SAFE_FUEL_OFF );
+        set_safe_fuel_thresh( 2.0 );
     } else {
         remove_flag( flag_SAFE_FUEL_OFF );
+        uilist tmenu;
+        tmenu.text = _( "Chose Safe Fuel Level Threshold" );
+        tmenu.addentry( 1, true, 'o', _( "Full Power" ) );
+        if (get_auto_start_thresh() < 0.80){
+            tmenu.addentry( 2, true, 't', _( "Above 80 %%" ) );
+        }
+        if (get_auto_start_thresh() < 0.55){
+            tmenu.addentry( 3, true, 'f', _( "Above 55 %%" ) );
+        }
+        if (get_auto_start_thresh() < 0.30){
+            tmenu.addentry( 4, true, 's', _( "Above 30 %%" ) );
+        }
+        tmenu.query();
+
+        switch( tmenu.ret ) {
+            case 1:
+                set_safe_fuel_thresh( 1.0 );
+                break;
+            case 2:
+                set_safe_fuel_thresh( 0.80 );
+                break;
+            case 3:
+                set_safe_fuel_thresh( 0.55 );
+                break;
+            case 4:
+                set_safe_fuel_thresh( 0.30 );
+                break;
+            default:
+                set_safe_fuel_thresh( 1.0 );
```

Comment:
```suggestion
```
The default should probably be to not edit the threshold, so that you don't set it to max every time you change your mind and close the menu before selecting something.